refarch-integrations/refarch-s3-integration/refarch-s3-integration-java/refarch-s3-integration-java-client-starter/src/main/java/de/muenchen/refarch/integration/s3/client/properties/S3IntegrationClientProperties.java

```diff
@@ -17,6 +17,6 @@ public class S3IntegrationClientProperties {
      */
     private DataSize maxBatchSize = DataSize.ofBytes(0L);
     /** Supported file extensions. */
-    private SupportedFileExtensions supportedFileExtensions;
+    private Map<String, String> supportedFileExtensions;
```

Comment:
Why is the data type altered here (again). Won't comment any further on this for now.

Reply:
LooseCouping warning